opentelemetry-jaeger/README.md

```diff
@@ -167,6 +167,7 @@ fn main() -> Result<(), Box<dyn std::error::Error + Send + Sync + 'static>> {
         .with_agent_endpoint("localhost:6831")
         .with_service_name("my_app")
         .with_tags(vec![KeyValue::new("process_key", "process_value")])
+	.with_max_packet_size(65_000)
```

Comment:
This default maximum only supports IPv6 and probably only on LANs. Can we assume the Jaeger agent is (almost) always addressed via IPv6, over a LAN?

Reply:
Good point, may be better to have a lower max in the examples 9216 byes is also common